cmd/generate.go

```diff
@@ -259,7 +258,7 @@ func getDependencies(path string, terragruntOptions *options.TerragruntOptions)
 			}
 
 			depPath := dep
-			terrOpts, _ := options.NewTerragruntOptions(depPath)
+			terrOpts, _ := options.NewTerragruntOptionsWithConfigPath(depPath)
```

Comment:
They refactored that function: since v0.48.7 `options.NewTerragruntOptions()` takes no parameters and returns a default configuration. To keep the backward compatibility the authors decided to create `			options.NewTerragruntOptionsWithConfigPath(path string)` which would mimic the original behaviour.